helm_resource/Tiltfile

```diff
@@ -71,7 +71,7 @@ def helm_resource(
 
   for i in range(len(image_deps)):
     key = image_keys[i]
-    if type(key) == 'str':
+    if type(key) == 'string':
```

Comment:
Mistake?

Reply:
:grimacing: 

```
nick@dopey:~/src/tilt$ starlark
Welcome to Starlark (go.starlark.net)
>>> type('x')
"string"
>>> ^D

nick@dopey:~/src/tilt$ python
Python 3.9.4 (default, Apr 15 2021, 18:12:50) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> type('x')
<class 'str'>
```

we just didn't notice this before because almost all helm charts use the repo/tag split.

Reply:
WHY STARLARK